src/components/util/document-store.tsx

```diff
@@ -0,0 +1,39 @@
+import { connectWallet, getWalletNetwork } from "./wallet";
+
+export const getDocumentStores = async (): Promise<Array<Record<string, unknown>>> => {
```

Comment:
how do you test? It doesn't look like it work at all for me.

My wallet: https://ropsten.etherscan.io/address/0xB26B4941941C51a4885E5B7D3A1B861E54405f90

the document store retrieved
![image](https://user-images.githubusercontent.com/503185/126130022-8032b293-5a4c-4614-8f4d-cca074024c33.png)

All the document stores are less than 63 days old + I didn't create that many in the last 2 months

Reply:
i found 100 contract creation log for "0xB26B4941941C51a4885E5B7D3A1B861E54405f90" wallet

Reply:
If you search 0xcae (first in the screenshot) You wont find it in the link you shared. I also can't find a document store:

![image](https://user-images.githubusercontent.com/503185/126587009-f94faea0-bf06-419e-a415-d70263ef931f.png)

You can try with my wallet if you want. The information are shared in a spreadsheet

Reply:
found the issue. fixed.

Reply:
![image](https://user-images.githubusercontent.com/503185/129837482-f11ed537-c1c5-4212-a4d9-809cf6b1d360.png)

I have no document store anymore

Reply:
<img width="1219" alt="Screenshot 2021-08-18 at 1 01 29 PM" src="https://user-images.githubusercontent.com/17894339/129840016-45bd1b7e-22ef-4b95-8567-faa653fa374f.png">

mmm weirdly it works for mine.

Reply:
ok I see some doc store, but I can't find https://ropsten.etherscan.io/address/0x8Fc57204c35fb9317D91285eF52D6b892EC08cD3

what about u ?

Reply:
0x8Fc57204c35fb9317D91285eF52D6b892EC08cD3 seems to be a very different from the current contract, it is using the older contract version. 

Based on my current code, it is currently retrieving the contracts that are of the current version, this feature will cause an issue if there is an upgrade on the upcoming/future contracts refactoring. i believe it is not worth continuing the feature of getting a document store as there isn't a simpler way to retrieve the past document store.

If u are okay with it, I will remove the retrieval of the document store and combo box but keep the document store validation just by checking the hashes length. This MR is getting huge, it will be preferable if i could get this merged into master before I change it to the localStorage implementation.